rpc/openapi/openapi.yaml

```
@@ -813,7 +813,38 @@ paths:
       tags:
         - Info
       description: |
-        Get genesis.
+        Get genesis document in a paginated/chunked format to make it
+        easier to iterate through larger gensis structures.
+      parameters:
+        - in: query
+          name: chunkID
+          description: Sequence number of the chunk to download.
+          schema:
+            type: integer
+            default: 0
+            example: 1
+      responses:
+        "200":
+          description: Genesis results.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GenesisChunkedResponse"
+        "500":
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+
+  /genesis_chunked:
```

```suggestion
  /genesis
```

then the route above would be _chunked?